readium/lcp/src/main/java/org/readium/r2/lcp/LcpService.kt

```diff
@@ -89,6 +111,11 @@ interface LcpService {
     fun contentProtection(authentication: LcpAuthenticating = LcpDialogAuthentication()): ContentProtection =
         LcpContentProtection(this, authentication)
 
+    /**
+     * Builds a [PublicationAsset] to open a LCP-protected publication from its license file.
+     */
+    fun remoteAssetForLicense(license: File): Try<PublicationAsset, LcpException>
```

Comment:
The license might not be stored on the file system, so I think we need a variant with maybe `ByteArray`?

Reply:
Not for now because it couldn't be updated through a ByteArray.